pages/common/cp.md

```diff
@@ -1,27 +1,35 @@
 # cp
 
-> Copy files.
+> Copy files and folders.
 
-- Copy files in arbitrary locations:
+- Copy a file to another location:
 
-`cp {{/path/to/original}} {{/path/to/copy}}`
+`cp {{/path/to/file.ext}} {{/path/to/copy.ext}}`
 
-- Copy a file to a parent directory:
+- Copy a file into another folder, keeping the filename:
```

Comment:
As I said in the opening comment, I've added both examples "to clarify when the target argument defines the actual copy, and when it defines the folder where to put the copy". Don't you think the second usage may not be clear if we don't explicitly mention it?